app/code/Magento/StoreGraphQl/Model/Resolver/Store/StoreConfigDataProvider.php

```diff
@@ -55,23 +65,42 @@ public function __construct(
      */
     public function getStoreConfigData(StoreInterface $store): array
     {
-        return array_merge(
-            $this->getBaseConfigData($store),
-            $this->getExtendedConfigData((int)$store->getId())
-        );
+        $defaultStoreConfig = $this->storeConfigManager->getStoreConfigs([$store->getCode()]);
+        return $this->prepareStoreConfigData(current($defaultStoreConfig), $store->getName());
     }
 
     /**
-     * Get base config data
+     * Get available website stores
      *
-     * @param StoreInterface $store
+     * @param string $websiteId
      * @return array
      */
-    private function getBaseConfigData(StoreInterface $store) : array
+    public function getAvailableStoreConfig(string $websiteId): array
```

Comment:
The same as already mentioned earlier.
```suggestion
    public function getAvailableStoreConfig(int $websiteId): array
```